docs/api/server/Environment-Variables.md

```diff
@@ -2,6 +2,542 @@
 
 # Environment Variables
 
-Documentation Forthcoming
+One App can be configured via Environment Variables:
 
-[☝️ Return To Top](#environment-variables)
\ No newline at end of file
+**By Topic**
```

Comment:
Shouldn't this be `## 📖 Table of Contents`

Reply:
Fixed in cf746a2